sycl/include/sycl/info/device_traits.def

```diff
@@ -198,6 +198,8 @@ __SYCL_PARAM_TRAITS_SPEC(device, ext_oneapi_bfloat16, bool,
 
 //Deprecated oneapi/intel extension
 //TODO:Remove when possible
+__SYCL_PARAM_TRAITS_SPEC(device, ext_intel_pci_device_id, pi_uint32,
+                         PI_DEVICE_INFO_PCI_DEVICE_ID)
```

Comment:
No need! Just add the `pci_device_id` and it should be good enough. 😄

Reply:
Removed.